edx_proctoring/api.py

```diff
@@ -605,19 +606,22 @@ def get_exam_attempt_data(exam_id, attempt_id, is_learning_mfe=False):
     low_threshold_pct = proctoring_settings.get('low_threshold_pct', .2)
     critically_low_threshold_pct = proctoring_settings.get('critically_low_threshold_pct', .05)
 
-    allowed_time_limit_mins = attempt.get('allowed_time_limit_mins') or 0
+    time_limit_mins = attempt.get('allowed_time_limit_mins') or 0
```

Comment:
any reason to change this variable? It seems better to stick with the name of the model attribute we're referencing `a_loud_time_limit_mins`

Reply:
👍🏻

Reply:
@ViktorRusakov ^^

Reply:
yes, no need to change, reverted back in 7e065d0